packages/nc-gui/components/smartsheet/expanded-form/Comments.vue

```diff
@@ -117,6 +117,10 @@ const saveComment = async () => {
   } finally {
     isSaving.value = false
   }
+
+  nextTick(() => {
+    commentInputDomRef.value?.focus()
+  })
```

Comment:
search `const focus: VNodeRef` in other cell components. I think the focus in Comment.vue is a bit overcomplicated.

Reply:
I tried that but not working, in comment.vue I'm disabling input when isSaving=true and we want focus back after comment saved

Reply:
```ts
const focusCommentInput: VNodeRef = (el) => {
  if (!isEditing.value || isExpandedFormCommentMode.value) {
    isExpandedFormCommentMode.value = false
    return (el as HTMLInputElement)?.focus()
  }
}
```

I tried above and commented following code:
```ts
// watch(commentInputDomRef, () => {
//   if (commentInputDomRef.value && isExpandedFormCommentMode.value) {
//     setTimeout(() => {
//       commentInputDomRef.value?.focus()
//       isExpandedFormCommentMode.value = false
//     }, 400)
//   }
// })
```

All thing are working fine like when I click comment option from cell context menu it will focus commentInput but when I tried to `Edit` comment and after saving this, `commentInput` is focusing again

If this is correct behaviour then let me know

Reply:
And also when I click expand row button at first `CommentInput` is focusing and then 1st input field from cell fields 
<img width="918" alt="Screenshot 2023-12-28 at 10 34 07 AM" src="https://github.com/nocodb/nocodb/assets/101566080/1c0f4e2b-4aad-4e27-b1dd-f62b7ee72647">

Reply:
Will take a look later.

Reply:
@rameshmane7218 Just tried a bit. I think it is a desired behaviour.

Reply:
Okay

Reply:
Fixed